src/pkg/shub/client/api.go

```diff
@@ -78,6 +78,9 @@ func getManifest(uri ShubURI) (manifest ShubAPIResponse, err error) {
 
 	// Do the request, if status isn't success, return error
 	res, err := httpc.Do(req)
+	if res == nil {
+		return ShubAPIResponse{}, fmt.Errorf("No response received from singularity hub")
```

Comment:
Why using `fmt` instead of `sylog`?

Reply:
he is returning an error type, to be handled else were, on the error handling step, it will go into a sylog.